renku/core/commands/graph.py

```diff
@@ -510,3 +512,26 @@ def workflow_has_identical_subprocesses(workflow_, subprocesses_ids_):
                     return workflow.association.plan
 
         return run
+
+
+@pass_local_client(requires_migration=True)
+def build_graph(client, revision, no_output, paths):
+    """"""
```

Comment:
It's still complaining 😄 : `D103: Missing docstring in public function`

Reply:
Yea I just noticed it in the tests. Pushed the fix right now :see_no_evil: